src/jahia2wp.py

```diff
@@ -41,6 +41,8 @@
   jahia2wp.py veritas               <csv_file>                      [--debug | --quiet]
   jahia2wp.py fan-global-sitemap    <csv_file> <wp_path>            [--debug | --quiet]
   jahia2wp.py inventory             <path>                          [--debug | --quiet]
+  jahia2wp.py search-and-replace-many <csv_file> <source> <destination> [--debug | --quiet]
+  jahia2wp.py search-and-replace-by-inventory <path> <source> <destination> [--debug | --quiet]
```

Comment:
Suggestion mais sur "release", on a une commande "backup-inventory" donc peut-être que tu pourrais renommer en "search-and-replace-inventory" ?

Reply:
OK